src/python_testing/TC_DRLK_2_9.py

```diff
@@ -202,15 +209,21 @@ async def get_user(self, userindex, username, useruniqueid, credentiallist, expe
             asserts.assert_true(response.userUniqueID == useruniqueid,
                                 "Error when executing GetUserResponse command, userUniqueID={}".format(
                                     str(response.userUniqueID)))
-            asserts.assert_true(response.credentials == credentiallist,
-                                "Error when executing GetUserResponse command, credentials={}".format(
-                                    str(response.credentials)))
-            asserts.assert_equal(expected_status, Status.Success)
+            logging.info("Credentials value is GetUserResponse Command %s" % (str(response.credentials)))
+            # traverse  through input credentials and match each value with the resonse credential
+            for input_credential_index in range(len(credentiallist)):
+                match_found = False
+                for response_credential_index in range(len(response.credentials)):
```

Comment:
While this loop now successfully allows credentials in the response to be in different order, it does not fully verify the test case: Need to add a check that the length of credentialList is equal to the length of response.credentials (to detect if response.credentials possibly has extra unexpected entries)

Reply:
Added the check though test plan does not mention this check. Will update the issue with this check

Reply:
Hmm, interesting point. Since user is created by this TC, there should not be any unexpected/additional credentials.